code/__DEFINES/directions.dm

```diff
@@ -0,0 +1,27 @@
+//Directions (already defined on BYOND natively, purely here for reference)
+/// define purely for readability, cables especially need to use this as `NO_DIRECTION` represents a "node"
+#define NO_DIRECTION 0
+//#define NORTH		1
+//#define SOUTH		2
+//#define EAST		4
+//#define WEST		8
+//#define NORTHEAST	5
+//#define SOUTHEAST 6
+//#define NORTHWEST 9
+//#define SOUTHWEST 10
+
+
+
+///Flips a direction along the horizontal axis, will convert E -> W, W -> E, NE -> NW, SE -> SW, etc
+#define FLIP_DIR_HORIZONTALLY(dir) (dir ^ (EAST|WEST))
+///Flips a direction along the vertical axis, will convert N -> S, S -> N, NE -> SE, SW -> NW, etc
+#define FLIP_DIR_VERTICALLY(dir) (dir ^ (NORTH|SOUTH))
```

Comment:
Some explanation on how these work would be nice, similar to IS_DIR_DIAGONAL

Reply:
done